main.nf

```diff
@@ -571,6 +572,7 @@ if(params.aligner == 'star'){
             --readFilesCommand zcat \\
             --runDirPerm All_RWX \\
             --outFileNamePrefix $prefix $seqCenter \\
+        samtools index *.bam
```

Comment:
Maybe can be more specific? I think `samtools index ${$prefix}.bam` should work...